slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c

```diff
@@ -2605,8 +2605,8 @@ int readSample(int ns) {
             // if channel is in enable mask
             if ((1 << ich) & (transceiverMask)) {
 
-                int offset = FIFO_TIN_STATUS_FIFO_EMPTY_1_OFST + ich;
-                uint32_t mask = (1 << offset);
+                // int offset = FIFO_TIN_STATUS_FIFO_EMPTY_1_OFST + ich;
```

Comment:
What is happening here?

Reply:
every transceiver channel (4 in total) have a bit to represent if its fifo is empty or not.